static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -1,21 +1,46 @@
-# How to update your pipeline by editing dvc files
+# Updating pipelines
 
-> 1)using `dvc move` to edit file paths of outputs or dependencies.
-> 2)editting by hand the outputs paths.
-> 3)`dvc run` to make edits in the dependency files and `dvc commit` to save
-> changes.
+If you need to rename or relocate the file paths and/or dependencies of a
+specific DVC-file, then you can 
 
-If your source file is a data file, then use dvc move:
+- use `dvc move` to edit file paths of outputs or dependencies.
+- `dvc run` to make edits in the files and/or dependencies and `dvc commit` to
+  save changes.
+
+If your source file is a data file, then use `dvc move`:
 
 `dvc move` is used if a file or a directory has been added to DVC with `dvc 
 add`. The source file or directory is moved to its destination path, renamed 
 and corresponding DVC-file is updated.
 
-If your source file isn't data file then use `dvc run`. Then add the new file
-with `dvc add`. If you check your file using `git mv` then it shows that your
-path isn't updated. So, using `git mv` is not an option.
+First add DVC files to get tracked by DVC file-tree. Use `dvc add data.dvc`
```

Comment:
?? I'm not sure I understand the first sentence here

Reply:
move is not permitted for stages that are not data sources. ``dvc move`` can only relocate data files and their paths and outputs. If the file is a .json or .dvc move can't relocate them.